.github/workflows/ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Continuous Integration
+on: [push, pull_request]
```

Comment:
I think we want either:

```suggestion
on:
  push:
    branches: [ main ]
  pull_request:
    branches: [ main ]
```

or:

```suggestion
on: [pull_request]
```

The current method runs every check twice: once for the push onto the branch, and a second time for the pull request. (see screenshot)

![Screen Shot 2022-11-21 at 10 47 59 AM](https://user-images.githubusercontent.com/41651655/203098204-6fdb46fb-1703-4f32-adeb-e64397ae106e.png)

The first proposed method will eliminate the `push` checks except for pushes to `main`. We should avoid pushing directly to main when possible, but if we ever need to then having the check couldn't hurt and would alert us to any issues. I think I _slightly_ prefer the first option for that reason, but I'm ok with either.

See:
- [GitHub Actions - Building and Testing Node.js](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs#using-the-nodejs-starter-workflow)
- [fqm-execution ci.yml](https://github.com/projecttacoma/fqm-execution/blob/0506a98e9313c618b3905b8bbdf7b037266bddf6/.github/workflows/ci.yml#L2_

Reply:
Good catch - I originally had the second option but added the `push` check to get things running on this PR and I forgot to remove it. I agree that it's probably better to use the first option for the reason you mentioned, so I can make that change